src/scenarios/shared/crossgen.py

```diff
@@ -0,0 +1,199 @@
+'''
+Module for running Crossgen
```

Comment:
nit: this doesn't run Crossgen.
```suggestion
Module for parsing Crossgen args
```